wingtips-zipkin2-spring-boot/src/main/java/com/nike/wingtips/springboot/zipkin2/WingtipsZipkinProperties.java

```diff
@@ -0,0 +1,76 @@
+package com.nike.wingtips.springboot.zipkin2;
+
+import com.nike.wingtips.zipkin2.WingtipsToZipkinLifecycleListener;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * A {@link ConfigurationProperties} companion for {@link WingtipsWithZipkinSpringBootConfiguration} that allows you to
+ * specify the configuration of {@link WingtipsToZipkinLifecycleListener} via your Spring Boot application's properties
+ * files. The following properties are supported (NOTE: {@code wingtips.zipkin.base-url} is required - all others are
+ * optional and can be left out, however it's highly recommended that you set {@code wingtips.zipkin.service-name} as
+ * well):
+ * <ul>
+ *     <li>
+ *         wingtips.zipkin.zipkin-disabled - Disables registering {@link WingtipsToZipkinLifecycleListener} with
```

Comment:
probably not great idea to change, but I think in boot it is conventional to just end in "enabled"

Reply:
If I had a time machine I'd go back and do it right the first time, but yeah, probably not a good idea to change now. :)